python/py-flatbuffers/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-flatbuffers
+# version should be the same as devel/flatbuffers
+version             1.12.0
+license             Apache-2
+platforms           darwin linux
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+description         Python runtime library for use with the Flatbuffers \
+                    serialization format.
+long_description    {*}${description}
+
+homepage            https://google.github.io/flatbuffers/
+
+python.versions     38
+
+if {${name} ne ${subport}} {
+
+    depends_build-append \
+                    port:flatbuffers \
+                    port:py${python.version}-setuptools
+
+    # Do not try to fetch or checksum anything as we will be using the
+    # Python flatbuffers source already present in the flatbuffers port
+    fetch           {}
+    checksum        {}
```

Comment:
Instead of overriding the fetch and checksum phases, if you want to have no distfiles, the way to do it is just:
```
    distfiles
```
(i.e., clear the distfiles variable).

Reply:
OK thanks, good to know